client/app/src/time/TimeService.js

```diff
@@ -0,0 +1,35 @@
+(function() {
+  'use strict';
+
+  angular.module('arkclient')
+    .service('timeService', [TimeService]);
+
+  /**
+   * TimeService
+   * @constructor
+   */
+  function TimeService() {
+    function __getJSON(url) {
+      return new Promise(function(resolve, reject) {
+        var xhr = new XMLHttpRequest();
```

Comment:
any specific reason to not use $http instead of xhr?

Reply:
No, in this case its the same.. its just my Habit. Should i Change?

Reply:
yes, please. it's better to keep the same standard as other services